README.markdown

```diff
@@ -135,6 +135,34 @@ Workers can be given multiple queues (a "queue list") and run on
 multiple machines. In fact they can be run anywhere with network
 access to the Redis server.
 
+#### Rails example
+
+If you installed the Resque as a Rails gem, the above example will look like this:
```

Comment:
I think this should worded differently: resque is always going to be installed as a gem, and there's no such thing as a 'rails gem'. Maybe something like "If you are using ActiveJob here's how your job definition will look:"